runtime/lua/vim/lsp.lua

```diff
@@ -2519,13 +2519,22 @@ function lsp.with(handler, override_config)
   end
 end
 
+--- @deprecated
+--- *Deprecated: Use |vim.lsp.inlay_hint.toggle()| instead.*
+---
 --- Enable/disable/toggle inlay hints for a buffer
----@param bufnr (integer) Buffer handle, or 0 for current
----@param enable (boolean|nil) true/false to enable/disable, nil to toggle
+---
+--- @param bufnr (integer) Buffer handle, or 0 for current
+--- @param enable (boolean|nil) true/false to enable/disable, nil to toggle
 function lsp.inlay_hint(bufnr, enable)
-  return require('vim.lsp.inlay_hint')(bufnr, enable)
+  vim.deprecate('vim.lsp.inlay_hint()', 'vim.lsp.inlay_hint.toggle()', '0.11')
+  return require('vim.lsp.inlay_hint').toggle(bufnr, enable)
 end
```

Comment:
Don't deprecate. Remove this completely.

Reply:
removed